http/oidc/src/test/java/org/wildfly/security/http/oidc/KeycloakConfiguration.java

```diff
@@ -60,8 +63,13 @@ public class KeycloakConfiguration {
      * </ul>
      */
     public static RealmRepresentation getRealmRepresentation(final String realmName, String clientId, String clientSecret,
-                                                             String clientHostName, int clientPort, String clientApp) {
-        return createRealm(realmName, clientId, clientSecret, clientHostName, clientPort, clientApp);
+                                                             String clientHostName, int clientPort, String clientApp, boolean configureClientScopes) {
+        RealmRepresentation realmRepresentation = createRealm(realmName, clientId, clientSecret, clientHostName, clientPort, clientApp);
+        if (configureClientScopes) {
```

Comment:
This won't block getting this merged, but since the scope needs to be added to the web app client specifically, instead of using `realmRepresentation.getClients().get(0)`, the `configureClientScopes` variable could be passed to `createRealm` so that this gets configured specifically when calling `createWebAppClient`.

Reply:
fixed!